src/grafana_dashboards/metrics-dashboard.json

```diff
@@ -292,131 +424,141 @@
       "datasource": {
         "uid": "${prometheusds}"
       },
+      "description": "",
       "fieldConfig": {
         "defaults": {
-          "decimals": 0,
-          "links": [],
+          "color": {
+            "fixedColor": "green",
+            "mode": "fixed"
+          },
+          "decimals": 2,
           "mappings": [],
           "thresholds": {
             "mode": "absolute",
             "steps": [
               {
                 "color": "green",
                 "value": null
-              },
-              {
-                "color": "yellow",
-                "value": 0
               }
             ]
           },
-          "unit": "short"
+          "unit": "bytes"
         },
         "overrides": []
       },
       "gridPos": {
-        "h": 4,
-        "w": 3,
-        "x": 14,
-        "y": 1
+        "h": 6,
+        "w": 5,
+        "x": 10,
+        "y": 6
       },
-      "id": 48,
-      "interval": "",
+      "id": 80,
       "options": {
         "colorMode": "background",
-        "graphMode": "none",
+        "graphMode": "area",
         "justifyMode": "auto",
         "orientation": "auto",
         "reduceOptions": {
           "calcs": [
-            "last"
+            "lastNotNull"
           ],
           "fields": "",
           "values": false
         },
+        "text": {},
         "textMode": "value"
       },
-      "pluginVersion": "9.2.1",
+      "pluginVersion": "9.5.3",
       "targets": [
         {
           "datasource": {
             "uid": "${prometheusds}"
           },
-          "expr": "cortex_prometheus_notifications_sent_total{juju_application=~\"$juju_application\",juju_model=~\"$juju_model\",juju_model_uuid=~\"$juju_model_uuid\",juju_unit=~\"$juju_unit\"}",
-          "format": "time_series",
-          "instant": true,
-          "interval": "",
-          "legendFormat": "",
+          "editorMode": "code",
+          "expr": "sum(rate(loki_ingester_chunk_stored_bytes_total[15m]))",
+          "range": true,
           "refId": "A"
         }
       ],
-      "title": "Alerts Event Total",
+      "title": "Logs growth rate per second",
```

Comment:
The panel should have a "units" property where you could select a rate. That way you wouldn't need to specify "per second" in the title.